docs/source/development-guide/style-guide/naming-conventions.rst

```diff
@@ -14,8 +14,8 @@ The general filename convention is as follows::
     imap_<instrument>_<datalevel>_<descriptor>_<startdate>_<enddate>_<version>.<extension>
 
 * ``<instrument>`` is the IMAP instrument associated with the data product. Acceptable values include: ``codice``,
-  ``glows``, ``hi-45``, ``hi-90``, ``hit``, ``idex``, ``lo``, ``mag``, ``swapi``, ``swe``, ``ultra-45``, and
-  ``ultra-90``
+  ``codice-hi``, ``codice-lo``, ``glows``, ``hi-45``, ``hi-90``, ``hit``, ``idex``, ``lo``, ``mag``, ``swapi``, ``swe``,
```

Comment:
From meeting: Should be one instrument "codice" with further distinctions in the descriptor